openapi/openapi.yaml

```diff
@@ -1647,6 +1647,11 @@ components:
           example: "9.0"
           description: Nvidia CUDA version which the application uses.
           nullable: true
+        platform:
+          type: string
```

Comment:
Is there a standard we follow with these strings?

Reply:
We use https://docs.python.org/3/library/sysconfig.html#sysconfig.get_platform to respect https://www.python.org/dev/peps/pep-0425/#platform-tag (distutils are too old, so sysconfig could be the right replacement).